### PR TITLE
Rework customer descriptions

### PR DESCRIPTION
**Pull request by noon-dawg**
## Summary

This pull request reworks the customer and customer timeline descriptions, and related files, to conform to the [writing style](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md).

## Links

N/A

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/paths/customers@{id}@edd-timeline.yaml [hunk 1]
```diff
@@ -41,21 +41,21 @@ post:
...
   tags:
     - Customers
-  summary: Create an EDD Timeline comment
+  summary: Create an EDD timeline comment
   operationId: PostEddTimeline
   x-sdk-operation-name: createEddTimelineComment
-  description: |
-    Create an EDD Timeline comment or custom defined event.
+  description: |-
+    Creates an EDD timeline comment or custom defined event.
```
Comment (Boydbueno) [inline openapi/paths/customers@{id}@edd-timeline.yaml:48]: It is not possible to create custom defined events with this endpoint
```suggestion
    Creates an EDD timeline comment.
```

File: openapi/components/schemas/Edd/Edd.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ properties:
   score:
     $ref: ./EddData.yaml
   nextUpdateTime:
-    description: The EDD score next update time.
+    description: Date and time at which the next EDD score is scheduled.
```
[the remark this answers is not in the document]
Comment (noon-dawg) [reply openapi/components/schemas/Edd/Edd.yaml:19]: Looks good!

File: openapi/components/schemas/Timelines/CustomerTimeline.yaml [hunk 3]
```diff
@@ -82,16 +82,16 @@ properties:
...
       - transaction-waiting-gateway
   customEventType:
-    description: >-
-      Timeline custom event type. Used with `custom-event` type. Must be defined
-      using [Customer Timeline custom event
-      API](#operation/PostCustomerTimelineCustomEventType).
+    description: |-
+      Timeline custom event type. Used with `custom-event` type. This
+      value must be defined using the _Create customer timeline custom
+      event type_ operation.
```
[the remark this answers is not in the document]
Comment (noon-dawg) [reply openapi/components/schemas/Timelines/CustomerTimeline.yaml]: Added in [3788b03](https://github.com/Rebilly/api-definitions/pull/1031/commits/3788b03a75801c3c09938d1eeacc0263b95497de)

It took me longer than it should have to figure out how to do this 😅 

([You can check out here](https://preview.redoc.ly/openapi-api/rework-descriptions-7/tag/Customers-timeline/operation/PostCustomerTimeline#!path=customEventType&t=request))

**Review by Boydbueno — DISMISSED** [review]

**Review by Boydbueno — DISMISSED** [review]

**Review by Boydbueno — DISMISSED** [review]

**Review by Boydbueno — DISMISSED** [review]

File: openapi/components/schemas/CurrencyCode.yaml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 type: string
 description: |-
-  ISO 4217 currency code.
+  Currency code in ISO 4217 format. Examples: `USD`, `EUR`, `CNY`.
```
Comment (adamaltman) [inline openapi/components/schemas/CurrencyCode.yaml:3]:
```suggestion
  Currency code in ISO 4217 format.
```
The docs generate examples from defined examples already. We should generally not include examples in description.

File: openapi/components/schemas/CustomerInformation.yaml [hunk 5]
```diff
@@ -1,21 +1,22 @@
 type: object
 properties:
   currency:
-    description: Currency (three letter ISO 4217 alpha code). For example, USD, EUR.
+    description: |-
+      Currency code in ISO 4217 format. Examples: `USD`, `EUR`, `CNY`.
```
Comment (adamaltman) [inline openapi/components/schemas/CustomerInformation.yaml:5]:
```suggestion
      Currency code in ISO 4217 format.
    minLength: 3
    maxLength: 3
    example: USD
```

File: openapi/components/schemas/SubscriptionSummaryMetrics.yaml [hunk 6]
```diff
@@ -1,17 +1,18 @@
 type: object
 properties:
   currency:
-    description: Currency (three letter ISO 4217 alpha code).
+    description: |-
+      Currency code in ISO 4217 format. Examples: `USD`, `EUR`, `CNY`.
```
Comment (adamaltman) [inline openapi/components/schemas/SubscriptionSummaryMetrics.yaml:5]:
```suggestion
      Currency code in ISO 4217 format.
    minLength: 3
    maxLength: 3
    example: USD
```

**Review by adamaltman — CHANGES_REQUESTED** [review]

File: openapi/components/schemas/Edd/Edd.yaml [hunk 7]
```diff
@@ -16,7 +16,7 @@ properties:
   score:
     $ref: ./EddData.yaml
   nextUpdateTime:
-    description: The EDD score next update time.
+    description: Date and time at which the EDD score is updated.
```
Comment (adamaltman) [inline openapi/components/schemas/Edd/Edd.yaml:19]:
```suggestion
    description: Date and time in ISO 8601 format when the EDD score is updated.
```

**Review by adamaltman — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**